cmd/scollector/doc.go

```diff
@@ -137,9 +137,43 @@ to at a 5 minute poll interval.
 	[[SNMP]]
 	  Community = "com"
 	  Host = "host"
+	  MIBs = ["cisco"]
 	[[SNMP]]
 	  Community = "com2"
 	  Host = "host2"
+	  # List of mibs to run for this host. Default is built-in set of ["ifaces","cisco"]
+	  MIBs = ["custom", "ifaces"]
+
+MIBs (map of string to table): Allows user-specified, custom snmp configurations.
+
+	[[MIBs]]
+	  [MIBS.cisco] #can name anything you want
+        BaseOid = "1.3.6.1.4.1.9.9" # common base for all metrics in this mib
```

Comment:
Whitespace and tabs are messed up for this whole doc block. Run it through something?